### PR TITLE
adding first draft of the metrics grabbing instructions

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist:

Please tick the following before making your PR:

* [x] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [x] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

**Review by supersven — APPROVED** [review]
LGTM

Thanks a lot for writing this up! :+1:

File: src/how-to/administrate/users.rst [hunk 1]
```diff
@@ -276,3 +276,42 @@ In order to obtain client logs on the iOS Wire client, follow this procedure:
 * Click on "Send report to wire"
 * A menu will open to share the debug report via email, allowing you to send it to the Wire team.
 
+How to retrieve metric values manually
```
Comment (fisx) [inline src/how-to/administrate/users.rst:279]: does this really belong into "administrate users", though?

Comment (arthurwolf) [reply src/how-to/administrate/users.rst:279]: The page as it is displayed in the menu is named "Investigative tasks", which I think this matches.
Maybe the page needs a filename change?
Or do you have a better idea of where to put this? (it's own page one level up?)

Comment (arthurwolf) [reply src/how-to/administrate/users.rst:279]: Screenshot from 2022-01-28 10-17-46

Comment (fisx) [reply src/how-to/administrate/users.rst:279]: you're right, the UI works, maybe the file could be renamed, but i'm not sure how urgent that is.

never mind me, thanks for the clarification!  :)

**Pull request merged**